src/components/advanced-table/advanced-table.tsx

```diff
@@ -153,8 +155,35 @@ export function AdvancedTable<
     setMaxPage(newPage);
   };
 
+  const onLastPage = page === maxPage && nextToken === undefined;
+
   const height = props.height ?? 'auto';
 
+  /**
+   * Renders the label to the left of the pagination buttons.
+   */
+  const labelDisplayedRows = useCallback(
+    ({ from, to, count }: LabelDisplayedRowsArgs) => {
+      if (count === -1) {
+        const loadedRows = rows?.length ?? 0;
+        if (onLastPage) {
+          // for some reason `to` is set incorrectly on the last page in
+          // server-side pagination, so we need a custom template literal for
+          // this case.
+          return `${from}-${loadedRows} of ${loadedRows}`;
```

Comment:
This should use `trans.__()` for localization.

Reply:
I don't think `trans.__()` works for dynamic strings like this. To my recollection, it's implemented naively as a dictionary mapping English strings to their localized counterparts. There's no way that would work with dynamic strings, since you would need a key for every possible string that can be generated from this template literal.

I suppose we could translate the word "of", but that brings its own localization difficulties.

I'd say for now we just leave this as-is for now until we find a better localization implementation that can handle this case.

Reply:
`trans_('%1–%2 of %3', from, to, total)`

Reply:
TY! I apologize for not doing more research on my own here.